app/helpers/quadicon_helper.rb

```diff
@@ -216,6 +221,48 @@ def render_quadicon_label(item, row)
     end
   end
 
+  # Renders a quadicon for PhysicalServer
+  #
+  def render_physical_server_quadicon(item, options)
+    output = []
+    if settings(:quadicons, :physical_server)
+      output << flobj_img_simple("layout/base.png")
+
+      output << flobj_p_simple("a72", (item.host ? 1 : 0))
+      output << flobj_img_simple("svg/currentstate-#{h(item.power_state.downcase)}.svg", "b72")
```

Comment:
Waiting for https://github.com/ManageIQ/manageiq/pull/14488 to be merged to resolve
```Error caught: [ActionView::Template::Error] undefined local variable or method `vendor' for #<ManageIQ::Providers::Lenovo::PhysicalInfraManager::PhysicalServer:0x007fb965178178>```